druid-shell/src/platform/windows/keyboard.rs

```diff
@@ -610,6 +610,8 @@ impl KeyboardState {
                     let code = scan_to_code(scan_code);
                     let key = if self.stash_utf16.is_empty() && wparam < 0x20 {
                         vk_to_key(vk).unwrap_or_else(|| self.get_base_key(vk, mods))
+                    } else if wparam == 0x7f {
```

Comment:
I think I would prefer `(wparam < 0x20 || wparam == 0x7f)` two lines up instead of this, as we already have the mapping and that would make it more likely that the non-ctrl and ctrl keys map to the same thing, in edge cases. But yes, this is the right place to fix it.

Reply:
@raphlinus I incorporated this, should I squash these commits?